mkbootimg.c

```diff
@@ -380,6 +560,14 @@ int main(int argc, char **argv)
 
             memcpy(hdr.magic, BOOT_MAGIC, BOOT_MAGIC_SIZE);
 
+            if((pagesize != 2048) && (pagesize != 4096)
+                && (pagesize != 8192) && (pagesize != 16384)
+                && (pagesize != 32768) && (pagesize != 65536)
+                && (pagesize != 131072)) {
+                fprintf(stderr, "error: unsupported pagesize %lu\n", pagesize);
```

Comment:
if you change `%lu` to `%u` then the code will compile! otherwise
`mkbootimg.c:567:64: error: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 3 has type ‘uint32_t’ {aka ‘unsigned int’} [-Werror=format=]
  567 |              fprintf(stderr, "error: unsupported pagesize %lu\n`

good job though!

Reply:
Thanks! I will fix that.

Reply:
> ကုဒ် ပြောင်း `%lu`ရင် compile လုပ်ပါလိမ့်မယ်။ `%u`မဟုတ်ရင် `mkbootimg.c:567:64: error: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 3 has type ‘uint32_t’ {aka ‘unsigned int’} [-Werror=format=] 567 | fprintf(stderr, "error: unsupported pagesize %lu\n`
> 
> အလုပ်ကောင်းတယ်!

Reply:
> ကျေးဇူးတင်ပါတယ်! ငါ အဲဒါကို ပြင်မယ်။